Modules/_datetimemodule.c

```diff
@@ -6653,6 +6681,7 @@ _datetime_exec(PyObject *module)
     seconds_per_day = PyLong_FromLong(24 * 3600);
     if (us_per_ms == NULL || us_per_second == NULL ||
         us_per_minute == NULL || seconds_per_day == NULL) {
+        PyMem_Free(capi);
```

Comment:
And at this point, the capsule is owned by the module object (because `PyModule_AddObject` stole the reference to it), so it should be cleaned up along with in the module object. Neither `PyMem_Free` nor DECREFing the capsule is necessary (here nor below).

Reply:
You are right. Updated.